todo.sh

```diff
@@ -217,9 +217,13 @@ actionsHelp()
 		      Displays all tasks that contain TERM(s) sorted by priority with line
 		      numbers.  Each task must match all TERM(s) (logical AND); to display
 		      tasks that contain any TERM (logical OR), use
-		      "TERM1\|TERM2\|..." (with quotes), or TERM1\\\|TERM2 (unquoted).
+		      'TERM1\|TERM2\|...' (with quotes), or TERM1\\\|TERM2 (unquoted).
```

Comment:
Why the quote change? Is it important?

Reply:
Not directly; for the following pipe character, it doesn't matter. But for other characters (dollar, backslash, backtick, ...) it does matter.
Also, I imagine this information is most important to beginners, who maybe use the Linux command-line just for todo.sh and not much else (yet). As single quoting has definite benefits (no doubling of backslashes) for the following (added) basic regexp escaping, I think it's best to just stick to single-quoting consistently here, or the alternative no-quoting, but not introduce a third alternative with double-quoting. That would just be too much for the beginner, who's just looking for an easy answer to get his searches working.